members/M000303.yaml

```diff
@@ -4,6 +4,8 @@ contact_form:
   action: ""
   steps:
     - visit: "http://www.mccain.senate.gov/public/index.cfm/contact-form"
+    - wait:
+      - value: 7
```

Comment:
does adding the 7 second wait make the submission work instead of throwing the 500 error?

Reply:
Unfortunately, no. I still get the 500 error when testing the yaml. In LogEntries, however, I get a Success message after a few minutes for the same yaml.

Reply:
in that case, we shouldn't make this change, since we don't want to unnecessarily lengthen the selenium sessions for each submission attempt

Reply:
you might want to test this change with the otus branch that has the blacklisting for the gif

Reply:
Will do. Which file is that one? I looked in the OTUS branch but wasn't sure where it was. Could you sent me the link/address?

Reply:
i believe Tom deployed it to yamltron integration. just change the yamltron tester url to that version

Reply:
Just tested it with that branch and got the same error. Any other ideas?